js/coinex.js

```diff
@@ -1822,6 +1824,100 @@ module.exports = class coinex extends Exchange {
+        //             "bkr_price_imply": "0.00000000000000000000",
+        //             "close_left": "0.0005",
+        //             "create_time": 1651202571.320778,
+        //             "deal_all": "19.72000000000000000000",
+        //             "deal_asset_fee": "0.00000000000000000000",
+        //             "fee_asset": "",
+        //             "finish_type": 1,
+        //             "first_price": "39441.12",
+        //             "insurance": "0.00000000000000000000",
+        //             "latest_price": "39441.12",
+        //             "leverage": "3",
+        //             "liq_amount": "0.00000000000000000000",
+        //             "liq_order_price": "0",
+        //             "liq_order_time": 0,
+        //             "liq_price": "16491.28560000000000011090",
+        //             "liq_price_imply": "0.00000000000000000000",
+        //             "liq_profit": "0.00000000000000000000",
+        //             "liq_time": 0,
+        //             "mainten_margin": "0.005",
+        //             "mainten_margin_amount": "0.09860280000000000000",
+        //             "maker_fee": "0.00000000000000000000",
+        //             "margin_amount": "11.57352000000000000000",
+        //             "market": "BTCUSDT",
+        //             "open_margin": "0.58687582908396110455",
+        //             "open_margin_imply": "0.00000000000000000000",
+        //             "open_price": "39441.12000000000000000000",
+        //             "open_val": "19.72056000000000000000",
+        //             "open_val_max": "19.72056000000000000000",
+        //             "position_id": 65171206,
+        //             "profit_clearing": "-0.00986028000000000000",
+        //             "profit_real": "-0.00986028000000000000",
+        //             "profit_unreal": "0.00",
+        //             "side": 2,
+        //             "stop_loss_price": "0.00000000000000000000",
+        //             "stop_loss_type": 0,
+        //             "sys": 0,
+        //             "take_profit_price": "0.00000000000000000000",
+        //             "take_profit_type": 0,
+        //             "taker_fee": "0.00000000000000000000",
+        //             "total": 3464,
+        //             "type": 1,
+        //             "update_time": 1651202638.911212,
+        //             "user_id": 3620173
+        //         },
+        //         "message":"OK"
+        //     }
+        //
+        const status = this.safeString (response, 'message');
+        const type = (addOrReduce === 1) ? 'add' : 'reduce';
+        return this.extend (this.parseModifyMargin (response, market), {
+            'amount': this.parseNumber (amount),
+            'type': type,
+            'status': status,
+        });
+    }
+
+    parseModifyMargin (data, market = undefined) {
+        return {
+            'info': data,
+            'type': undefined,
+            'amount': undefined,
```

Comment:
Can we derive the `amount` from `data` ?

Reply:
There doesn't seem to be a corresponding amount field in the data